From 6d33152766177db191516896f64feefb77acad85 Mon Sep 17 00:00:00 2001
From: Harry Wentland <harry.wentland@amd.com>
Date: Tue, 9 Feb 2016 16:19:06 -0500
Subject: [PATCH 1353/1565] drm/amd/amdgpu: Fall back to non-dal driver on
 Bonaire

Since DAL support for Bonaire isn't ready yet don't try
to enable DAL for Bonaire yet.

Signed-off-by: Harry Wentland <harry.wentland@amd.com>
Reviewed-by: Alex Deucher <alexander.deucher@amd.com>
---
 drivers/gpu/drm/amd/amdgpu/amdgpu_device.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_device.c b/drivers/gpu/drm/amd/amdgpu/amdgpu_device.c
index 1e04231..98fdc93 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_device.c
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_device.c
@@ -1380,11 +1380,6 @@ static int amdgpu_resume(struct amdgpu_device *adev)
 bool amdgpu_device_has_dal_support(struct amdgpu_device *adev)
 {
 	switch(adev->asic_type) {
-#if defined(CONFIG_DRM_AMD_DAL) && defined(CONFIG_DRM_AMD_DAL_DCE8_0)
-	case CHIP_BONAIRE:
-	case CHIP_HAWAII:
-		return true;
-#endif
 #if defined(CONFIG_DRM_AMD_DAL) && defined(CONFIG_DRM_AMD_DAL_DCE11_0)
 	case CHIP_CARRIZO:
 		return true;
-- 
1.9.1

